README.md

```diff
@@ -192,6 +192,31 @@ Since keys are unique there is no way to map values to keys using `Colors.value(
 Inheriting from a `Ruby::Enum` class, all defined enums in the parent class will be accessible in sub classes as well.
 Sub classes can also provide extra enums as usual.
 
+``` ruby
+class A
+  include Ruby::Enum
+
+  define :T, 'T'
```

Comment:
Nit, maybe we can have an example with colors like above, or something that mimics reality? I personally get lost in A's, T's and Us :)